packages/framework/fluid-framework/package.json

```diff
@@ -105,6 +105,38 @@
       },
       "EnumDeclaration_ConnectionState": {
         "forwardCompat": false
+      },
```

Comment:
Similar here, please check to see if this was expected breaking change.

Reply:
these seem to be in next already: https://github.com/microsoft/FluidFramework/blob/next/packages/framework/fluid-framework/package.json

Reply:
This is different set. Related to Josh's PR.